lib/arclight/custom_document.rb

```diff
@@ -31,5 +34,34 @@ def to_solr(solr_doc = {})
     def names
       [corpname, famname, name, persname].flatten.compact.uniq - repository
     end
+
+    # Combine subjets into one group from:
+    #  <controlaccess/><subject></subject>
+    #  <controlaccess/><function></function>
+    #  <controlaccess/><genreform></genreform>
+    #  <controlaccess/><occupation></occupation>
+    def access_subjects(subjects = [])
```

Comment:
Is there a case where we pass in an array of subjects to the `access_subjects` method?  (I couldn't find any looking through the code briefly)